packages/auth-construct/src/construct.ts

```diff
@@ -900,132 +902,209 @@ export class AmplifyAuth
       Stack.of(this)
     )
   ): void => {
```

Comment:
Since this file is already quite large, this method could be a good candidate to break out into another file. Not a blocker though